templates/components/streamfield/blocks/accordion.html

```diff
@@ -1,5 +1,29 @@
 {% verbatim %}
-<div class="">
-    {{ self }}
+{% load wagtailcore_tags %}
+
+<div class="container mx-auto my-16 text-black">
+    {% for accordion in accordions %}
+    <details class="max-w-3xl my-12 border-b border-grey-300">
+        {# webkit-details-marker hides the default arrow on safari #}
+        <summary
+            class="flex flex-col my-4 gap-2 hover:cursor-pointer focus-visible:outline-0 [&::-webkit-details-marker]:hidden">
+            <span class="font-semibold text-3xl hover:underline">{{ accordion.title }}</span>
+            {# Show/Hide Button - Extra group focus classes to wrap focus around link instead of full block #}
+            <div
+                class="text-base text-mackerel-300 hover:text-mackerel-200 focus-visible:outline focus-visible:outline-2 focus-visible:outline-mackerel-300 focus-visible:w-fit mb-8">
```

Comment:
The hover text color has very low contrast:

<img width="336" alt="image" src="https://github.com/user-attachments/assets/b6396040-65a3-488c-9146-fd8308052002" />

The non-hovered color could use a bit more too:

<img width="336" alt="image" src="https://github.com/user-attachments/assets/47249495-7f25-471f-912d-27048cb30fb7" />

Reply:
We could switch the styles so that they are`text-mackerel-400 hover:text-mackerel-300`, which solves the problem of the hover text being too light but not the issue of the main text needing a bit more contrast. If it needs a bit more here then it likely needs a bit more in other places.

Perhaps the best way forward for the scope of this PR is to make the `text-mackerel-400 hover:text-mackerel-300` change and then revisit whether the global styles need some adjustment for accessibility purposes?